### PR TITLE
i18n(es): Translate `bun.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `bun.mdx`

### CODE REVIEW

File: src/content/docs/es/recipes/bun.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+bun ./dist/server/entry.mjs
+```
+
+## Recursos Oficiales
+
+- [Construye una aplicación con Astro y Bun](https://bun.sh/guides/ecosystem/astro)
+
+## Recursos de la Comunidad
+
+¿Usando Bun con Astro? ¡Agrega tu entrada de blog o video a esta página!
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/bun.mdx:108]:
```suggestion
¿Estás usando Bun con Astro? ¡Agrega tu entrada de blog o video a esta página!
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**